shopfloor_mobile_base/static/wms/src/components/list.js

```diff
@@ -184,3 +199,15 @@ Vue.component("list-item", {
     </div>
   `,
 });
+
+Vue.component("list-item-title-action", {
+    mixins: [ItemDetailMixin],
+    template: `
+<v-btn icon class="detail-action" :class="{'theme--dark': opts.theme_dark}" link
+        v-if=""
+        @click="opts.on_title_action ? opts.on_title_action(record): on_detail_action(record, opts.title_action_field)">
+    <btn-info-icon size="small" v-if="!opts.title_action_icon"/>
+    <v-icon v-if="opts.title_action_icon">{{ opts.title_action_icon }}</v-icon>
+</v-btn>
```

Comment:
not sure I understand, why aren't you using the `show_title_action()` and `title_action_position()` methods?

Reply:
Because they are not on this component. I render this component IF they say so. I forgot the empty v-if apparently